lib/web_ui/test/ui/fallback_fonts_golden_test.dart

```diff
@@ -261,6 +282,11 @@ void testMain() {
       ]);
     });
 
+    // https://github.com/flutter/flutter/issues/157763
+    test('prioritizes Noto Color Emoji over Noto Sans Symbols', () async {
+      await checkDownloadedFamilyForCharCode(0x1f3d5, 'Noto Color Emoji');
```

Comment:
Should we also `expect()` that `Noto Sans Symbols 2` was NOT downloaded?

Reply:
Since this is a check for a single char, only one font should have been downloaded, so I'm going to check that "downloadedFonts.length" is 1.

I don't think there's any case where we download multiple fonts for a single character?

Reply:
> I don't think there's any case where we download multiple fonts for a single character?

Yes, that is the case, but it would be nice to have a test for it so it stays that way 🙂

> check that "downloadedFonts.length" is 1

SGTM!